### DISCUSSION TITLE
继续一个详细的使用教程啊

### DISCUSSION THREAD

**Discussion by wu5shen** [wenda-LLM/wenda#259] [Q&A]
哪位大佬录个视频详细介绍一下各个功能呗，最近更新跳跃很大啊，不知道怎么用了

**Comment by HHUtian** [comment]
希望哪位大佬能看到把，最近网上找资料，要崩溃了，对于小白来说，还是太难了，如果只是目前的教程，也只是能用而已，并不能完成想要的东西，急需多一些，详细的教程
